nf_core/components/info.py

```diff
@@ -296,14 +308,27 @@ def generate_component_info_help(self):
             outputs_table = Table(expand=True, show_lines=True, box=box.MINIMAL_HEAVY_HEAD, padding=0)
             outputs_table.add_column(":outbox_tray: Outputs")
             outputs_table.add_column("Description")
-            outputs_table.add_column("Pattern", justify="right", style="green")
+            if self.component_type == "modules":
+                inputs_table.add_column("Pattern", justify="right", style="green")
+            elif self.component_type == "subworkflows":
+                inputs_table.add_column("Structure", justify="right", style="green")
```

Comment:
because this in the following is basically the same, like the inputs table, can't we just have a generic `generate_params_table` function to reduce code duplication?